frontend/src/components/HelpPopup/HelpPopup.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+// import userEvent from "@testing-library/user-event";
+
+import HelpPopup from "./HelpPopup";
+import userEvent from "@testing-library/user-event";
+
+jest.mock(
+  "./HelpPopup",
+  () =>
+    function () {
+      return <span>HelpPopup</span>;
+    },
+);
+
+test("TID 3.1. Renders HelpPopup component", () => {
+  const dom = render(<HelpPopup />);
```

Comment:
An easier way is we can do 
```suggestion
  const { getByTestId, getByText } = render(<HelpPopup />);
```